tool/src/main/java/org/wildfly/security/tool/CredentialStoreCommand.java

```diff
@@ -87,6 +94,7 @@
     public static final String ITERATION_PARAM = "iteration";
     public static final String PASSWORD_CREDENTIAL_VALUE_PARAM = "secret";
     public static final String ADD_ALIAS_PARAM = "add";
+    public static final String ALIAS_PARAM = "alias";
```

Comment:
This is rather an argument name than a param (name). The other argument names aren't in constants.

Reply:
https://issues.redhat.com/browse/ELY-2094